build.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+####
+# For more documentation, see:
+#  https://github.com/OpenLiberty/application-stack/wiki/Open-Liberty-Application-Stack-Customization
+####
+
+#
+# Base image used to build stack image
+#
+BASE_OS_IMAGE="${BASE_OS_IMAGE:-adoptopenjdk/openjdk11-openj9:ubi}"
+
+#
+# Version of Open Liberty runtime to use within both inner and outer loops
+#
+OL_RUNTIME_VERSION="${OL_RUNTIME_VERSION:-20.0.0.12}"
+
+#
+# The Open Liberty base image used in the final stage of the outer loop Dockerfile used to build your application image from
+#
+OL_UBI_IMAGE="${OL_UBI_IMAGE:-openliberty/open-liberty:20.0.0.12-full-java11-openj9-ubi}"
+
+# The name and tag of the gradle stack image you will build.  This will used to create your inner loop development containers, and also as the base image for the first stage of your outer loop image build.
+#
+STACK_IMAGE="${STACK_IMAGE:-openliberty/application-stack-gradle:0.4}"
```

Comment:
Not sure it helps to try to match the maven version..it'll get out of sync soon enough.   Could be 0.1